mathics/builtin/__init__.py

```diff
@@ -186,6 +189,28 @@ def name_is_builtin_symbol(module, name: str) -> Optional[type]:
     return module_object
 
 
+def print_message(symbol_name: str, tag: str, *msgs, channel=stderr):
```

Comment:
We should not be adding more code to `builtin/__init__.py`  but instead should be removing it. 

It doesn't look like this is used anywhere. I would imagine it could be used in testing. Is it tested at all?  Inside a test routine is where it would belong.

Reply:
Regarding this, I initially wrote it as a default message function for the functions that require one as a parameter, but in the end, I used it just for doing tests.  I can remove it from this and put it in another branch in case we want to use it for low-level tests.

Reply:
Done. The function is now in https://github.com/Mathics3/mathics-core/tree/context_free_print_message, in `mathics.builtin.base`